src/schema/objects/common_principles.yaml

```diff
@@ -93,6 +93,9 @@ label:
     for example, it is `rest` in `task-rest` following `task-<label>` specification.
     Note that labels MUST not collide when casing is ignored
     (see [Case collision intolerance](SPEC_ROOT/02-common-principles.md#case-collision-intolerance)).
+    For a given data type, labels MUST be consistent across subjects and sessions.
+    For example label `acq-hires` for the `anat` data of a given subject
+    must mean the same thing as the label `acq-hires` used for the `anat` of any other subject.
```

Comment:
This is the main bit.

I am restricting "consistency" to within a datatype (anat, func, eeg...) because:
- most previous mentions of consistency appeared within the page of a given modality
- I could imagine that `acq-highRes` could easily mean different things in MRI and EEG for example.

Reply:
What is the level of consistency required? It seems like it's up to the curator. E.g., `acq-hires` could be 0.5x0.5x0.5 in one subject, and 0.6x0.6x0.6 in another, as long as there's some other anatomical scan in each that is lower-resolution, right? Or should they be differentiated in that case (e.g., `acq-hires` and `acq-`reallyhires`)?